client/ayon_speedtree/hooks/pre_subprocess_no_window.py

```diff
@@ -0,0 +1,21 @@
+import subprocess
+from ayon_applications import PreLaunchHook, LaunchTypes
+
+
+class LaunchServerNoWindow(PreLaunchHook):
+    """Specifically for Zbrush to make the AYON tools launching faster
```

Comment:
Is this really needed for SpeedTree too?

Also, we should really make sure that when launching in dev mode or from `ayon_console` or alike that at least the communicator's console is also a visible dialog OR that in some way or another you can access this process's logs because that really eases the ability to debug what's going on if anything fails.